coccinelle/glnx-errors.cocci

```diff
@@ -0,0 +1,6 @@
+@@
+expression p;
+@@
+- glnx_set_error_from_errno (p);
+- goto out;
++ return  glnx_throw_errno (p);
```

Comment:
One problem obviously is that coccinelle seems to hardcode linux kernel style (I didn't find an option to change it), and in general doesn't try to preserve "pretty" indentation.

So I'm not actually sure we should use this one, but it demos what we *could* do.